sklearn/utils/mocking.py

```diff
@@ -55,7 +58,7 @@ def fit(self, X, y):
     def predict(self, T):
         if self.check_X is not None:
             assert_true(self.check_X(T))
-        return T.shape[0]
+        return np.zeros(_num_samples(T))
```

Comment:
well yeah, but there is no ``classes_`` here.... and I think it would be overkill to add it. as we know what we pass to it in the  tests.